api.planx.uk/send/helpers.ts

```diff
@@ -1,16 +1,7 @@
-import AdmZip from "adm-zip";
 import fs from "fs";
 import { gql } from "graphql-request";
-import path from "path";
 import airbrake from "../airbrake";
-import { getFileFromS3 } from "../s3/getFile";
 import { adminGraphQLClient } from "../hasura";
-import {
-  hasRequiredDataForTemplate,
-  generateDocxTemplateStream,
-} from "@opensystemslab/planx-document-templates";
-import { $admin } from "../client";
-import { Passport } from "../types";
 
 /**
```

Comment:
These docstrings can be tidied up as well now that we've lost `downloadFile()` 🗑️